jans-cedarling/bindings/cedarling_python/src/config/bootstrap_config.rs

```diff
@@ -5,520 +5,160 @@
+        let inner = cedarling::BootstrapConfig::load_from_json(config_json)
+            .map_err(|e| PyValueError::new_err(e.to_string()))?;
+        Ok(Self {
+            inner,
+            source: None,
+        })
+    }
 
-        self.userinfo_iss_validation = "disabled".to_string();
-        self.userinfo_aud_validation = "disabled".to_string();
-        self.userinfo_sub_validation = "disabled".to_string();
-        self.userinfo_exp_validation = "disabled".to_string();
+    #[getter]
+    pub fn get_policy_store_local_fn(&mut self) -> Option<String> {
+        self.source
+            .as_ref()
+            .map(|s| s.policy_store_local_fn.clone())?
     }
-}
 
-fn get_required<'py, T>(options: &Bound<'py, PyDict>, key: &str) -> PyResult<T>
-where
-    T: FromPyObject<'py>,
-{
-    options
-        .get_item(key)?
-        .ok_or_else(|| PyKeyError::new_err(format!("Missing required config: `{}`", key)))?
-        .extract()
-}
+    #[setter]
+    pub fn set_policy_store_local_fn(&mut self, path: &str) -> PyResult<()> {
+        match self.source.as_mut() {
+            Some(source) => {
+                source.policy_store_local_fn = Some(path.to_string());
+                self.inner = cedarling::BootstrapConfig::from_raw_config(source).map_err(|e| PyValueError::new_err(e.to_string()))?;
+                Ok(())
+            }
+            None => Err(PyValueError::new_err("Failed to set `CEDARLING_POLICY_STORE_LOCAL_FN` since a policy store wasn't loaded yet.")),
+        }
+    }
 
-fn get_optional<'py, T>(options: &Bound<'py, PyDict>, key: &str) -> PyResult<Option<T>>
-where
-    T: FromPyObject<'py>,
-{
-    // let opt = options.get_item(key)?;
-    options.get_item(key)?.map(|x| x.extract()).transpose()
-}
+    #[getter]
+    pub fn get_log_type(&mut self) -> Option<String> {
+        self.source.as_ref().map(|s| s.log_type.to_string())
+    }
 
-fn get_with_default<'py, T>(options: &Bound<'py, PyDict>, key: &str, default: T) -> PyResult<T>
-where
-    T: FromPyObject<'py> + Clone,
-{
-    Ok(options
-        .get_item(key)?
-        .map(|x| x.extract())
-        .transpose()?
-        .unwrap_or(default))
-}
+    #[setter]
+    pub fn set_log_type(&mut self, log_type: &str) -> PyResult<()> {
+        match self.source.as_mut() {
+            Some(source) => {
+                source.log_type = LoggerType::from_str(log_type)
+                    .map_err(|e| PyValueError::new_err(e.to_string()))?;
+                self.inner = cedarling::BootstrapConfig::from_raw_config(source)
+                    .map_err(|e| PyValueError::new_err(e.to_string()))?;
+                Ok(())
+            },
+            None => Err(PyValueError::new_err(
+                "Failed to set `CEDARLING_LOG_TYPE` since a policy store wasn't loaded yet.",
+            )),
+        }
+    }
 
-impl TryFrom<BootstrapConfig> for cedarling::BootstrapConfig {
-    type Error = PyErr;
+    #[getter]
+    pub fn get_log_ttl(&mut self) -> Option<u64> {
```

Comment:
I don't think that we really need getters in this structure..

Reply:
getters removed in [3c383ea](https://github.com/JanssenProject/jans/pull/10350/commits/3c383eacce4deba2e1dd57e5e9f87c5496a2f67d)